test/2_management_api/correlations/get_by_process_model_id.js

```diff
@@ -38,15 +39,15 @@ describe('Management API:   GET  ->  /correlations/process_model/:process_model_
 
     const result = await testFixtureProvider
       .managementApiClientService
-      .startProcessInstance(testFixtureProvider.identities.defaultUser, processModelId, startEventId, payload, returnOn);
+      .startProcessInstance(identity, processModelId, startEventId, payload, returnOn);
 
     should(result).have.property('correlationId');
     should(result.correlationId).be.equal(payload.correlationId);
 
     return result.correlationId;
   }
 
-  it('should return a correlation by its id through the Management API', async () => {
+  it('should return a correlation by its id for an user through the Management API', async () => {
```

Comment:
```suggestion
  it('should return a correlation by its id for a user through the Management API', async () => {
```

Classic pitfall: https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user